libpod/runtime_ctr.go

```diff
@@ -71,6 +71,7 @@ func (r *Runtime) initContainerVariables(rSpec *spec.Spec, config *ContainerConf
 		ctr.config.ShmSize = DefaultShmSize
 	} else {
 		// This is a restore from an imported checkpoint
+		ctr.restoreFromCheckpoint = true
```

Comment:
Do we only need this once, for the first mount? Because this isn't stored in the DB so it won't persist after the first time Podman runs.

Reply:
I think this does not need to be in database. But I am not 100% sure.

Is there a scenario where additional mounts can be added to container after it is running?

From what I understood, once the root file-system is configured the mountlabel is stored in config.MountLabel. This is now correct for restored containers and not overwritten. From this point in time all additional mounts should behave just like for a newly started container. Currently I see no need to have the information (restoredFromCheckpoint) in the database.

Reply:
Mounts can not be added after the container is created, correct.

Reply:
It looks like this is only used in container initialization, not after, so we should be set.

Reply:
Agreed.